src/recipe.jl

```diff
@@ -110,6 +113,53 @@ function Makie.plot!(plot::Beeswarm)
         calculate!(point_buffer.val, algorithm, direction == :y ? pixelspace_point_buffer.val : reverse.(pixelspace_point_buffer.val), markersize, side)
         # Project the beeswarm back to data space and store it, again, in `point_buffer.val`
         point_buffer.val .= Point2f.(Makie.project.((scene.camera,), :pixel, :data, direction == :y ? (point_buffer.val) : reverse.(point_buffer.val)))
+
+        # Method to create a gutter when a gutter is defined
+        # NOTE: Maybe turn this into a helper function?
+
+        if !isnothing(plot.attributes[:gutter].val)
```

Comment:
This could be in pixelspace, so acting on `point_buffer` directly before line 115.  In this way we are guaranteed that the categorical variable is in the `x` dimension.  But that depends on what system of coordinates `gutter` is in.

Reply:
See my follow-up comment after latest push.